Software/tft/README.md

```diff
@@ -0,0 +1,17 @@
+# The Firmware Tool
+This is a simple tool to add header to raw firmware binary. The header is used by driver to check if hardware is compatible with the image. The header is only 16 bytes long and contains PCI Vendor ID, PCI Device ID and PCI Hardware Revision ID values to check. It also contains the CRC value of the raw image.
```

Comment:
This is a simple tool to add a header to a raw firmware binary. The header is used by a driver to check if the hardware is compatible with the image. The header is only 16 bytes long and contains a PCI Vendor ID, a PCI Device ID and a PCI Hardware Revision ID values to check. It also contains the CRC value of the raw image.